test/utils/validateMemos.test.ts

```diff
@@ -122,12 +130,16 @@ describe('validateMemos', function () {
     }
 
     // Good:
+    callValidateMemos([{ type: 'text', value: '' }], extraInfo)
     callValidateMemos([{ type: 'text', value: 'hello' }], extraInfo)
 
     // Bad:
     expect(() =>
       callValidateMemos([{ type: 'text', value: 'hello!!' }], extraInfo)
     ).throws('Memo Test Coin message: cannot be longer than 6 characters')
+    expect(() =>
+      callValidateMemos([{ type: 'hex', value: '1234' }], extraInfo)
+    ).throws('Memo Test Coin message: cannot be type hex')
```

Comment:
What about `callValidateMemos([{ type: 'text', value: '1234' }], extraInfo)`? That's more what I meant. Having a case where the data is hex but the type is specified as `'text'` could be good incase implementation incorrectly assumes because the data looks like hex that it will be assumed a hex type. Better regression coverage me thinks